warbler.gemspec

```
@@ -31,7 +31,7 @@ bundle up all of your application files for deployment to a Java environment.}
   gem.add_runtime_dependency 'rake', ['>= 10.1.0']
   gem.add_runtime_dependency 'jruby-jars', ['>= 9.0.0']
   gem.add_runtime_dependency 'jruby-rack', ['>= 1.1.1', '< 1.3']
-  gem.add_runtime_dependency 'rubyzip', ['~> 1.0', '< 1.4']
+  gem.add_runtime_dependency 'rubyzip', ['>= 1.0.0', '<= 2.3.0']
```

How about not bounding?

```suggestion
  gem.add_runtime_dependency 'rubyzip', '>= 1.0.0'
```